deps/llvm.mk

```diff
@@ -175,6 +175,19 @@ LLVM_CPPFLAGS += -flto
 LLVM_LDFLAGS += -flto
 endif # LLVM_LTO
 
+ifeq ($(BUILD_CUSTOM_LIBCXX),1)
+LLVM_LDFLAGS += -Wl,-R$(build_libdir)
+LLVM_CXXFLAGS += -I$(build_includedir)
+LLVM_CPPFLAGS += -I$(build_includedir)
+LLVM_CFLAGS += -I$(build_includedir)
```

Comment:
Do we really have to add `-I$(build_includedir)` to all of them? Can't we just put it into `CPPFLAGS`?  :panda_face: